hbase-rest/pom.xml

```diff
@@ -368,9 +370,17 @@
               <goal>compile</goal>
             </goals>
             <phase>generate-sources</phase>
+            <configuration>
+              <protocArtifact>com.google.protobuf:protoc:${internal.protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
I think the only difference is this configuration? We could introduce a property for this so in hbase-protocol-shade and hbase-reset, we use the new protoc compiler while in hbase-protocol, we use the old one?

Reply:
Yes, we could, but I think that the would hide the difference, and make it even harder to find which version we use where.

All the different protobuf libraries and compilers are hard enough figure out already, the current setup at least gives a clue to whoever tries to decypher it.